environments/vm/group_vars/vm.yml

```diff
@@ -439,7 +441,7 @@ manage:
         scopes: ["READ"]
       }
     - {
-        name: "sp-dashboard",
+        name: "{{ manage.sp_dashboard_user }}",
```

Comment:
Unfortunately, this will lead to a recursive loop problem.  Better use a separate variable, like sp_dashboard_manage_user or something